backend/molgenis-emx2-sql/src/main/java/org/molgenis/emx2/sql/MetadataUtils.java

```diff
@@ -103,21 +127,18 @@ private MetadataUtils() {
   /**
    * Returns version number. Returns -1 if metadata does not exist (i.e. MOLGENIS schema does not
    * exist)
-   *
-   * @param jooq
-   * @return
    */
   protected static Integer getVersion(DSLContext jooq) {
     if (jooq.meta().getSchemas(MOLGENIS).size() == 0) {
       // schema does not exist
       return -1;
     } else {
-      Result<org.jooq.Record> result = jooq.selectFrom(VERSION_METADATA).fetch();
+      Result<Record> result = jooq.selectFrom(VERSION_METADATA).fetch();
       if (result.size() > 0) {
-        Object version = result.get(0).get(VERSION);
+        Integer version = result.get(0).get(VERSION);
```

Comment:
If that's the case the migration would fail as well right? ->

```java
int version = MetadataUtils.getVersion(db.getJooq());
```

What was the previous version type?